src/components/ListItem.vue

```diff
@@ -32,32 +31,37 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex';
+// import { mapGetters } from 'vuex';
 
 export default {
   props: {
     listItem: {
       type: Object,
       default: () => ({}),
-      description: 'The list item was passed from List with more info like array of'+
-      'objects which each obj includes text and state key values',
+      description: 'The list item was passed from List with more info like array of'
```

Comment:
This description is misleading. What happens when this component is used outside of a List, or not as part of an array? The descriptions should have only what needs to be there for others to use. You should try to answer these questions:

- What is this prop's purpose?
- Does it require any other info than the type? (if it's object, it should say what it needs.)
- Are there any side effects of this prop?

Reply:
I added the following
` description: 'List Item is an object includes text and state key values'`